discovery_engine_core/ai/bert/src/pooler.rs

```diff
@@ -51,13 +51,17 @@ impl<const N: usize> From<[f32; N]> for Embedding<Ix1> {
     }
 }
 
+#[derive(Clone, Debug, Error)]
+#[error("bytes do not represent a valid embedding")]
+pub struct MalformedBytesEmbedding;
```

Comment:
🔧 since we already use `displaydoc::Display` in this crate, we can use it here as well
```suggestion
#[derive(Clone, Debug, Display, Error)]
/// Bytes do not represent a valid embedding
pub struct MalformedBytesEmbedding;
```